x-pack/legacy/plugins/maps/public/layers/styles/vector/vector_style.js

```diff
@@ -33,14 +33,31 @@ const POLYGONS = [GEO_JSON_TYPE.POLYGON, GEO_JSON_TYPE.MULTI_POLYGON];
 export class VectorStyle extends AbstractStyle {
 
   static type = 'VECTOR';
-  static STYLE_TYPE = { 'DYNAMIC': DynamicStyleProperty.type, 'STATIC': StaticStyleProperty.type };
+  static STYLE_TYPE = STYLE_TYPE;
+  static createDescriptor(properties = {}) {
+    return {
+      type: VectorStyle.type,
+      properties: { ...getDefaultProperties(), ...properties }
+    };
+  }
+
+  static createDefaultStyleProperties(mapColors) {
+    return getDefaultProperties(mapColors);
+  }
+
+  static getDisplayName() {
```

Comment:
Why was getDisplayName added? Where is it used?

Reply:
It wasn't. Some sort of oddity with github diffing.

Reply:
I see that now. Its not used, mind removing it?